apollo-api/build.gradle.kts

```diff
@@ -58,6 +60,10 @@ kotlin {
   }
 }
 
+tasks.withType<KotlinCompile> {
+  kotlinOptions.jvmTarget = "1.8"
+}
+
```

Comment:
Maybe put this in `apollo-android/build.gradle.kts` next to the others ? https://github.com/apollographql/apollo-android/blob/3e37125cc4f98e6e981c37e877c1cc4f61f17604/build.gradle.kts#L57